website/docs/r/universal_source.html.markdown

```diff
@@ -0,0 +1,64 @@
+---
+layout: "sumologic"
+page_title: "SumoLogic: sumologic_universal_source"
+description: |-
+  Provides a Sumologic Universal source.
+---
+
+# sumologic_universal_source
+Provides a [Sumologic Universal source][1].
```

Comment:
Actually we should not expose these source as "Universal" sources, but `Cloud-to-Cloud" sources. We don't actually mention "Universal" anywhere in the UI or help doc except in the JSON configurations.

Reply:
Changing it in the markdown is not sufficient, we need to replace universal with cloud-to-cloud everywhere in code, or at least the resource name (or anything user facing).